bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx126beta-en-GB.html

```diff
@@ -0,0 +1,51 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ #}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}What’s new with Firefox{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_126beta') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block wnp_content %}
+  <section class="wnp-content-main">
+    <div class="mzp-l-content mzp-t-content-sm">
+      <aside class="wnp-swap">
+        <div class="wnp-swap-blurb-here">
+          <span class="wnp-swap-text-here" lang="en">hello 🫖</span>
+          {% include 'firefox/whatsnew/includes/fx126beta/arrow-blue.svg' %}
+        </div>
+        <div class="wnp-swap-blurb-there">
+          <span class="wnp-swap-text-there" lang="fr">bonjour 🥖</span>
+          {% include 'firefox/whatsnew/includes/fx126beta/arrow-white.svg' %}
+        </div>
+      </aside>
+
+      <h2>Say ‘<span lang="fr">Bonjour</span>’ to Firefox&nbsp;translations</h2>
+
+      <p>Translate the web locally, so your data stays safe. Firefox now comes with an integrated translation tool, so you never have to use a third-party app again.</p>
+
+      <p>Click the {% include 'firefox/whatsnew/includes/fx126beta/translate-icon.svg' %} in Firefox’s address bar and enjoy content from around the world.</p>
```

Comment:
**Suggestion (a11y):** We should include accessible text for the icon here because we're clearly indicating this icon as part of the user's journey to translate the page. I notice the svg file has `aria-hidden=true` in it. Maybe make it `false` and add an `aria-label` to it?

Reply:
That was a carry-over from WNP118, but I agree we could make this a bit more accessible... with an inline SVG I think a `title` is read by screen readers, but we could also make it an inline image with alt, or maybe an image-replaced element with regular text...